### PR TITLE
UCP/PROTO/RNDV: Add performance data to rendezvous protocols

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Following up on #8282 , adding performance estimation information to rendezvous protocols

### CODE REVIEW

**Comment by yosefe** [comment]
@rakhmets @brminich @dmitrygx can you pls take a look?

File: src/ucp/rndv/proto_rndv.c [hunk 1]
```diff
@@ -331,7 +348,8 @@ ucs_status_t ucp_proto_rndv_rts_init(const ucp_proto_init_params_t *init_params)
         .unpack_time         = UCS_LINEAR_FUNC_ZERO,
         .perf_bias           = context->config.ext.rndv_perf_diff / 100.0,
         .mem_info.type       = init_params->select_param->mem_type,
-        .mem_info.sys_dev    = init_params->select_param->sys_dev
+        .mem_info.sys_dev    = init_params->select_param->sys_dev,
+        .ctrl_msg_name       = "RTS"
```
Comment (dmitrygx) [inline src/ucp/rndv/proto_rndv.c:352]: wdyt about adding `UCP_PROTO_RNDV_RTS_NAME`? - same as we have for `UCP_PROTO_RNDV_ATS_NAME`/`UCP_PROTO_RNDV_ATP_NAME`

File: src/ucp/rndv/rndv_rtr.c [hunk 2]
```diff
@@ -57,9 +58,11 @@ ucp_proto_rndv_rtr_common_init(const ucp_proto_init_params_t *init_params,
         .super.flags         = UCP_PROTO_COMMON_INIT_FLAG_RESPONSE,
         .remote_op_id        = UCP_OP_ID_RNDV_SEND,
         .unpack_time         = unpack_time,
+        .unpack_perf_node    = unpack_perf_node,
         .perf_bias           = 0.0,
         .mem_info.type       = mem_type,
         .mem_info.sys_dev    = sys_dev,
+        .ctrl_msg_name       = "RTR"
```
Comment (dmitrygx) [inline src/ucp/rndv/rndv_rtr.c:65]: `UCP_PROTO_RNDV_RTR_NAME`?

File: src/ucp/rndv/proto_rndv.h [hunk 3]
```diff
@@ -11,6 +11,10 @@
 
 #include <ucp/proto/proto_multi.h>
 
+/* Names of rendezvous acknowledgment messages */
```
Comment (dmitrygx) [inline src/ucp/rndv/proto_rndv.h:14]: [minor] add one more empty line? (e.g. like in [src/ucp/rndv/rndv_get.c](https://github.com/openucx/ucx/pull/8308/files#diff-ac996cf0c5ee1c54da915a335063af78605d944b4634970bfa3247d3cf3713cf))

File: src/ucp/rndv/proto_rndv.c [hunk 4]
```diff
@@ -193,20 +193,23 @@ ucp_proto_rndv_ctrl_init(const ucp_proto_rndv_ctrl_init_params_t *params)
...
     ucp_memory_info_t mem_info;
+    const char *rndv_op_name;
     ucs_status_t status;
     double ctrl_latency;
     uint16_t op_flags;
 
     ucs_assert(params->super.flags & UCP_PROTO_COMMON_INIT_FLAG_RESPONSE);
     ucs_assert(!(params->super.flags & UCP_PROTO_COMMON_INIT_FLAG_SINGLE_FRAG));
 
+    rndv_op_name                   = ucp_operation_names[params->remote_op_id];
```
[the remark this answers is not in the document]
Comment (yosefe) [reply src/ucp/rndv/proto_rndv.c:204]: It cannot fit into one line (when aligned by =)

**Review by brminich — APPROVED** [review]

**Review by dmitrygx — DISMISSED** [review]

**Review by rakhmets — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Pull request merged**